backend/wait-for-postgres.sh

```
@@ -0,0 +1,18 @@
+#!/bin/sh
+# wait-for-postgres.sh
+
+set -e
+
+host="$1"
+shift
+cmd="$@"
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)